clfd/apps/cleanup.py

```diff
@@ -2,7 +2,7 @@
 # NOTE: this is the right shebang, compatible with virtual / conda environments
 # https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take
 
-# NOTE: This script cannot be named clfd.py, because that makes any statement 
+# NOTE: This script cannot be named clfd.py, because that makes any statement
 # 'import clfd.X' fail
 
 import sys
```

Comment:
Line 40, command line argument --fmt: add "pfd" as a possible choice

Reply:
Just added it.